content/vendor/guides/production-ready.md

```diff
@@ -0,0 +1,194 @@
+
+### Helm
+
+Helm charts are supported by KOTS but not required. If for applications that are already packaged using helm, then the helm support in KOTS can help get an app packaged faster. If an application does not presently use helm, there's no requirement to use helm, as the KOTS built-in templating includes much of the same functionality, and the Admin Console includes a deep [kustomize.io integration](/vendor/kotsadm/updating/patching-with-kustomize) to greatly reduce the amount of templating required by app maintainers in the first place.
+
+### Operators
+Operators are good for specific use cases, we've written in-depth about them in our [Operators Blog Post](https://blog.replicated.com/operators-in-kots/). In general we recommend thinking deeply about the problem space an application solves before going down the operator path. They're really cool and powerful, but take a lot of time to build and maintain
+
+
+<!-- coming soon, wait for guide
+### Licensing and Entitlements
+  - how we think about it 
+  - basic enforcement
+  - advanced enforcement via API
+
+Basic license enforcement comes baked in for free -- expiration 
+
+Our recommendation is always to enforce licensing with a strong contract, but if you have strict licensing needs, you can always implement [Runtime License Validation](/vendor/guides/runtime-license-validation) in your application. This involves modifying your application code to query an API exposed by KOTS to consume the license payload, and validating the signature in the hot code paths in your application's core business logic.
+-->
+
+### Bundling and Analyzing Logs with Support bundle
+
+A robust support bundle is essential to minimizing back-and-forth when things go wrong. At a very minimum every app's support bundle should contain logs for an application's core pods. Usually this will be done with label selectors. To the labels for an application, either inspect the YAML, or run
+
+```shell
+kubectl get pods --show-labels
+```
+
+against a running instance to see what labels are used. Once the labels are discoved, a [logs collector](https://troubleshoot.sh/reference/collectors/pod-logs/) can be used to include logs from these pods in a bundle. Depending on the complexity of an app's labeling schema, you may need a few different declarations of the `logs` collector.
+
+As commmon issues are encountered in the field, it will make sense to add not only collectors but also analyzers to an apps's support stack. For example, when an error in a log file is discoved that should be surfaced to an end user user in the future, a simple [Text Analyzer](https://troubleshoot.sh/reference/analyzers/regex/) can detect specific log line in the future and inform an end user of remediation steps.
```

Comment:
```suggestion
As common issues are encountered in the field, it will make sense to add not only collectors but also analyzers to an apps's support stack. For example, when an error in a log file is discovered that should be surfaced to an end user in the future, a simple [Text Analyzer](https://troubleshoot.sh/reference/analyzers/regex/) can detect specific log line in the future and inform an end user of remediation steps.
```